com.makemkv.MakeMKV.yaml

```diff
@@ -0,0 +1,112 @@
+          url-template: https://ftp.gnu.org/gnu/wget/wget-$major.$minor.$patch.tar.gz
+  - name: x264
+    config-opts:
+      - --disable-cli
+      - --enable-shared
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        commit: baee400fa9ced6f5481a728138fed6e867b0ff7f
+  - name: ffmpeg
+    config-opts:
+      - --enable-gpl
+      - --disable-static
+      - --enable-shared
+      - --disable-doc
+      - --disable-ffplay
+      - --disable-ffprobe
+      - --enable-libopus
+      - --enable-libvpx
+      - --enable-libx264
+    cleanup:
+      - /share/ffmpeg/examples
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-5.1.2.tar.xz
+        sha256: 619e706d662c8420859832ddc259cd4d4096a48a2ce1eefd052db9e440eef3dc
+        x-checker-data:
+          type: anitya
+          project-id: 5405
+          stable-only: true
+          url-template: https://ffmpeg.org/releases/ffmpeg-$major.$minor.$patch.tar.xz
+  - name: makemkv-oss
+    buildsystem: autotools
+    build-options:
+      cflags: -D__GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -D __STDC_FORMAT_MACROS
+    config-opts:
+      - --enable-allcodecs
+    make-install-args:
+      # really just to avoid call to ldconfig
+      - DESTDIR=/
+    post-install:
+      - install -Dp -m 644 com.makemkv.MakeMKV.appdata.xml --target-directory=/app/share/appdata
+      - install -Dp -m 644 License.txt --target-directory=/app/share/licenses/com.makemkv.MakeMKV
+    sources:
+      - type: archive
+        url: https://www.makemkv.com/download/makemkv-oss-1.17.2.tar.gz
+        sha256: a83f8abb3f23def0dc8783e535062a75b7df2f7612291a8a83a21f90b93f2da4
+        x-checker-data:
+          type: html
+          url: https://makemkv.com/download
+          version-pattern: "MakeMKV v([\\d\\.]*)"
+          url-template: https://www.makemkv.com/download/makemkv-oss-$major.$minor.$patch.tar.gz
+      - type: file
+        path: com.makemkv.MakeMKV.appdata.xml
+  - name: makemkv-bin
+    buildsystem: simple
+    build-commands:
+      # TODO: put eula in appstream data once it supports it
+      # NOTE: author allows redistribution as long as eula is referenced in package description
+      - mkdir -p tmp
+      - echo accepted > tmp/eula_accepted
+      - make install DESTDIR=/ PREFIX=/app
+      - install -Dp -m 644 src/eula_en_linux.txt --target-directory=/app/share/licenses/com.makemkv.MakeMKV
+    sources:
+      - type: archive
+        url: https://www.makemkv.com/download/makemkv-bin-1.17.2.tar.gz
+        sha256: 80008ccc9ee864293f20d49e25a57b1a717d872ffa17d7744032e9e633e21789
+        x-checker-data:
+          type: html
+          url: https://makemkv.com/download
+          version-pattern: "MakeMKV v([\\d\\.]*)"
+          url-template: https://www.makemkv.com/download/makemkv-bin-$major.$minor.$patch.tar.gz
+    cleanup:
+      - /include
+      - /lib/debug
+      - /lib/pkgconfig
```

Comment:
Doesn't this now only apply to makemkv-bin? Apparently `/lib/debug` causes the error and removing it would allow moving this back to the top-level cleanup (see #44).

Reply:
I'll remove the debug line and move it back up to top-level. The issue should be resolved then. Yes.